modules/custom/az_marketing_cloud/templates/node--view--az-marketing-cloud--50-50-layout-image-right.html.twig

```diff
@@ -0,0 +1,118 @@
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ * - view_mode: View mode; for example, "teaser" or "full".
+ * - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+ * - page: Flag for the full page state. Will be true if view_mode is 'full'.
+ * - readmore: Flag for more state. Will be true if the teaser content of the
+ *   node cannot hold the main body content.
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_node()
+ *
+ * @ingroup themeable
+ */
+#}
+
+<table cellpadding="0" cellspacing="0" width="100%" role="presentation"
+  style="text-align: left; min-width: 100%; border-top: 1px solid #D4D4D4;"
+  class="stylingblock-content-wrapper">
+  <tbody>
+    <tr>
+      <td style="padding: 15px 20px;" class="stylingblock-content-wrapper" align="left">
+        <table cellspacing="0" cellpadding="0" role="presentation" class="qs-layout-5050" style="width: 100%;">
+          <tbody>
+            <tr dir="rtl">
+              <td valign="top" class="responsive-td" style="width: 50%; padding-right: 7.5px;">
+                <table cellpadding="0" cellspacing="0" width="100%" role="presentation"
+                  style="text-align: left; min-width: 100%; " class="stylingblock-content-wrapper">
+                  <tbody>
+                    <tr>
+                      <td class="stylingblock-content-wrapper" align="left">
+                        <h3 class="qs-heading" style="color:#403635; font-size:18px; font-style:normal; font-weight:bold; line-height:115%; margin-top:0;">
+                          {{ content.smart_title }}
+                        </h3>
+                        <p class="qs-summary" style="line-height: 150%;">
+                          {{ content.field_az_summary }}</p>
+                        <p>
+                          <a class="qs-link" href="{{ az_canonical_url }}"
+                              style="color:#8B0015; text-decoration:underline; font-weight:700; line-height:100%;">{{ az_canonical_summary_link_title }}</a>
+                        </p>
+                      </td>
+                    </tr>
+                  </tbody>
+                </table>
+              </td>
+              <td valign="top" class="responsive-td" style="width: 50%; padding-left: 7.5px;">
+                <table width="100%" cellspacing="0" cellpadding="0" role="presentation">
+                  <tbody>
+                    <tr>
+                      <td align="center">
+                        <img class="qs-img" src="{{ images.az_marketing_cloud_50_50_layout }}" alt="" width="270" style="display: block; height: auto; width: 100%;">
```

Comment:
@its-me-mario Changing to `width="270"` makes this inconsistent with the original 50/50 Layout (Image left). Is that okay?

Reply:
I would say yes.

The [original code snippets](https://codepen.io/itsmemario/pen/GRXNPRL) was using 200 for 30/70 and 270 for 50/50, so it must have been missed.

The image left should really be updated, but the width attribute is a fallback if the style value is broken in some way and I'm not aware of that being a use case for the email clients we support.

Reply:
Thank you! I'll make them both 270 in this PR and test it out.